### ISSUE TITLE
Error: 'insertBefore' of null in IE11

### ISSUE THREAD

**Issue by peio72** [bradberger/angular-material-calendar#36]
The calendar doesn' display on IE11: fires Error: 'insertBefore' of null.

Only Toolbar with Month and Nav is displayed.

**Comment by peio72** [comment]
Any chance this issue could be fixed or provide quick tip so I could fix it ? We have a live app using the calendar and need to work in IE.
The bug is still visible on the demo page browsing with IE and click on next month arrow.
Thanx to help asap.

**Comment by bradberger** [comment]
Sure, I'll look it over today. Sorry for the delay.

**Comment by gavinbarron** [comment]
I've taken a look at the demo page running IE 11.1000.14257.0 on Windows 10 Enterprise Build 14257 with no extensions or other variations from base and the calendar loads and works correctly for me.
Can you please provide some more information about the environment that you're seeing this behavior? 
Are you observing this on multiple machines? Do you have some corporate alterations to your build that might be interfering?

**Comment by peio72** [comment]
Yes Calendar loads but ... Did you click the next month button with developper tools opened ? You should see the error. This error prevents other parts of my js calendar  management to run.

**Comment by gavinbarron** [comment]
Had clicked through successfully and then opened dev tools, oops.

**Comment by peio72** [comment]
Can you see the errors now ?

**Comment by gavinbarron** [comment]
I can see the errors, it's something deep inside Angular that's throwing those errors.
You're more than welcome to contribute.

**Comment by peio72** [comment]
The new demo site is crashing my Firefox.

**Comment by julienmartin** [comment]
I got the same issue (AngularJS 1.5.5 + Angular Material 1.1 RC4). Fix planned?
